plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestGlobalTransactionMySqlConnectorTest.java

```diff
@@ -14,28 +14,19 @@
 package io.trino.plugin.mysql;
 
 import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.jdbc.BaseJdbcConnectorSmokeTest;
 import io.trino.testing.QueryRunner;
-import io.trino.testing.sql.SqlExecutor;
 
 import static io.trino.plugin.mysql.MySqlQueryRunner.createMySqlQueryRunner;
 
 public class TestGlobalTransactionMySqlConnectorTest
-        // TODO(https://github.com/trinodb/trino/issues/7019) define shorter tests set that exercises various read and write scenarios (a.k.a. "a smoke test")
-        extends BaseMySqlConnectorTest
+        extends BaseJdbcConnectorSmokeTest
```

Comment:
Should it be `TestGlobalTransactionMySqlConnectorSmokeTest` or `TestGlobalTransactionMySqlSmokeTest`?
There is `Smoke` in class name for Oracle.

Reply:
`TestMySqlGlobalTransactionMyConnectorSmokeTest`